### ISSUE TITLE
Unresolved identifier 'HTTPMetric' when using Firebase Performance

### ISSUE THREAD

**Issue by EugeneGordin2303** [firebase/firebase-ios-sdk#1071]
I'm trying to add FirebasePerformance to my app. For adding monitoring for specific network requests I used the example from here: https://firebase.google.com/docs/perf-mon/get-started-ios

However, I get "Unresolved identifier 'HTTPMetric' ". I do have imports for Firebase & FirebasePerformance in my .swift class.

I used pod 'Firebase/Performance' to add FirebasePerformance to my app. So I noticed this:

<img width="309" alt="screen shot 2018-04-11 at 12 21 27 pm" src="https://user-images.githubusercontent.com/9437985/38638394-faf8b89a-3d82-11e8-95e5-d1b1aa1f2253.png">

Basically I don't see HTTPMetric class in the framework. Is this a Beta issue? When should we expect to have a fix for this? Thank you

**Comment by morganchen12** [comment]
What are the contents of your `Podfile.lock`?

**Comment by EugeneGordin2303** [comment]
`PODS:
  - Alamofire (4.5.1)
  - AlamofireImage (3.3.0):
    - Alamofire (~> 4.5)
  - AlamofireNetworkActivityIndicator (2.2.0):
    - Alamofire (~> 4.5)
  - CocoaLumberjack/Core (3.0.0)
  - CocoaLumberjack/Default (3.0.0):
    - CocoaLumberjack/Core
  - CocoaLumberjack/Extensions (3.0.0):
    - CocoaLumberjack/Default
  - CocoaLumberjack/Swift (3.0.0):
    - CocoaLumberjack/Extensions
  - Crashlytics (3.9.3):
    - Fabric (~> 1.7.2)
  - Fabric (1.7.2)
  - Fingertips (0.5.0)
  - Firebase (4.5.0):
    - Firebase/Core (= 4.5.0)
  - Firebase/Core (4.5.0):
    - FirebaseAnalytics (= 4.0.4)
    - FirebaseCore (= 4.0.10)
  - Firebase/Performance (4.5.0):
    - Firebase/Core
    - FirebasePerformance (= 1.0.6)
  - FirebaseAnalytics (4.0.4):
    - FirebaseCore (~> 4.0)
    - FirebaseInstanceID (~> 2.0)
    - GoogleToolboxForMac/NSData+zlib (~> 2.1)
    - nanopb (~> 0.3)
  - FirebaseCore (4.0.10):
    - GoogleToolboxForMac/NSData+zlib (~> 2.1)
    - nanopb (~> 0.3)
  - FirebaseInstanceID (2.0.5)
  - FirebasePerformance (1.0.6):
    - FirebaseAnalytics (~> 4.0)
    - FirebaseInstanceID (~> 2.0)
    - GoogleToolboxForMac/Logger (~> 2.1)
    - GoogleToolboxForMac/NSData+zlib (~> 2.1)
    - GTMSessionFetcher/Core (~> 1.1)
    - Protobuf (~> 3.1)
  - GoogleToolboxForMac/Defines (2.1.3)
  - GoogleToolboxForMac/Logger (2.1.3):
    - GoogleToolboxForMac/Defines (= 2.1.3)
  - GoogleToolboxForMac/NSData+zlib (2.1.3):
    - GoogleToolboxForMac/Defines (= 2.1.3)
  - GTMSessionFetcher/Core (1.1.14)
  - nanopb (0.3.8):
    - nanopb/decode (= 0.3.8)
    - nanopb/encode (= 0.3.8)
  - nanopb/decode (0.3.8)
  - nanopb/encode (0.3.8)
  - OCMock (3.4.1)
  - PMKVObserver (2.0.2):
    - PMKVObserver/ObjC (= 2.0.2)
    - PMKVObserver/Swift (= 2.0.2)
  - PMKVObserver/ObjC (2.0.2)
  - PMKVObserver/Swift (2.0.2):
    - PMKVObserver/ObjC
  - Protobuf (3.5.0)
  - SAMKeychain (1.5.3)
  - SideMenu (3.1.4)
  - SSZipArchive (2.1.1)
  - SwiftyJSON (4.0.0)
  - ZendeskSDK (1.11.2.1):
    - ZendeskSDK/Providers (= 1.11.2.1)
    - ZendeskSDK/UI (= 1.11.2.1)
  - ZendeskSDK/Providers (1.11.2.1)
  - ZendeskSDK/UI (1.11.2.1):
    - ZendeskSDK/Providers

DEPENDENCIES:
  - Alamofire (~> 4.5)
  - AlamofireImage (~> 3.1)
  - AlamofireNetworkActivityIndicator (~> 2.2)
  - CocoaLumberjack/Swift (~> 3.0.0)
  - Crashlytics
  - Fabric
  - Fingertips
  - Firebase
  - Firebase/Performance
  - OCMock
  - PMKVObserver (~> 2.0)
  - SAMKeychain
  - SideMenu
  - SSZipArchive
  - SwiftyJSON
  - ZendeskSDK

SPEC CHECKSUMS:
  Alamofire: 2d95912bf4c34f164fdfc335872e8c312acaea4a
  AlamofireImage: 2e784dc5d00f04903a52c1d169181469c805c3df
  AlamofireNetworkActivityIndicator: 81e1630e7cfed22a4e4e82a174c59e651dc4f8ca
  CocoaLumberjack: c823149bccc5519a9447aeb433be7b1212a7d6a5
  Crashlytics: dbb07d01876c171c5ccbdf7826410380189e452c
  Fabric: 9cd6a848efcf1b8b07497e0b6a2e7d336353ba15
  Fingertips: 8e557eed25253fcb4bfa25ac4418f8e8add1ad9a
  Firebase: d83d287bdb6b1c78320561482706d1fcfc8ddfc1
  FirebaseAnalytics: 722b53c7b32bfc7806b06e0093a2f5180d4f2c5a
  FirebaseCore: da5e8e372cb6046212739a4d81e42d32c22907cc
  FirebaseInstanceID: f2b688c66b972f30d7fa9f5f9f91455454a03b47
  FirebasePerformance: fa032c27e229eb8c1a8638918793fe2e47465205
  GoogleToolboxForMac: 2501e2ad72a52eb3dfe7bd9aee7dad11b858bd20
  GTMSessionFetcher: 390ea358e5a0d0133153806f744662dad933d06b
  nanopb: 5601e6bca2dbf1ed831b519092ec110f66982ca3
  OCMock: 2cd0716969bab32a2283ff3a46fd26a8c8b4c5e3
  PMKVObserver: d6d7ff5eb11c049d3f51194576e6c4aa2588d107
  Protobuf: 8a9838fba8dae3389230e1b7f8c104aa32389c03
  SAMKeychain: 483e1c9f32984d50ca961e26818a534283b4cd5c
  SideMenu: 99021e70b3f2eee65b5dd88e4274d09031b1221e
  SSZipArchive: 14401ade5f8e82aba1ff03e9f88e9de60937ae60
  SwiftyJSON: 070dabdcb1beb81b247c65ffa3a79dbbfb3b48aa
  ZendeskSDK: 2cda4db2ba6b10ba89aeb8dddaa94e97c85946a0

PODFILE CHECKSUM: a059bc7e811401e4a61336878f7297b57ee2be70

COCOAPODS: 1.3.1
`

**Comment by morganchen12** [comment]
Does running `pod update` make the issue go away?

**Comment by EugeneGordin2303** [comment]
oh wow, it actually does! thank you!
